build/pmm-client-docker/Dockerfile-alpine

```diff
@@ -0,0 +1,34 @@
+FROM alpine:latest
+
+ENV MONIT_VERSION=5.25.2
+
+# Compile and install monit
+RUN apk add --update make gcc curl zlib-dev libressl-dev musl-dev \
+    && cd /tmp \
+    && curl -sS https://mmonit.com/monit/dist/monit-${MONIT_VERSION}.tar.gz | gunzip -c - | tar -xf - \
+    && cd monit-${MONIT_VERSION} \
+    && ./configure --without-pam \
+    && make \
+    && make install
```

Comment:
Isn't monit already in alpine repo? Why to compile it? https://pkgs.alpinelinux.org/packages?name=monit&branch=v3.8

Reply:
reworked, thanks